assets/src/room_ui.ts

```diff
@@ -72,6 +72,12 @@ export function getVideoButtonStatus(): boolean {
   return videoButton.dataset.enabled === "true";
 }
 
+
+const animateClick = (el:HTMLButtonElement) => {
+el.classList.add("animate")
+setTimeout(() => el.classList.remove("animate"), 500)
```

Comment:
The purpose of the animation is to let user know when the button is clicked/tapped, especially on mobile. There is no hover effect on mobile so I thought it will be good fix. Thank you for suggestion to style `:active` pseud-class, I implemented it.